app/models/country.rb

```diff
@@ -0,0 +1,3 @@
+class Country < ApplicationRecord
+  validates :title, presence: true, length: { minimum: 3, maximum: 25 }
```

Comment:
change maximum to 100
![image](https://user-images.githubusercontent.com/70864999/154537214-59a90b55-8a54-47d7-8b7e-01c921ee7780.png)